saleor/graphql/schema.graphql

```diff
@@ -3300,12 +3307,9 @@ type ProductVariant implements Node {
   sku: String!
   product: Product!
   trackInventory: Boolean!
-  quantityAllocated: Int!
```

Comment:
AFAIK We should not drop fields in API in that way. We should mark them as a deprecated with the info that we will drop them in the next release

Reply:
Ok, we can discuss during the daily how to handle this properly

Reply:
Yep we cannot remove fields from the schema like that, we should deprecate them and say in which Saleor version it will be removed ( in this case 2.11 - two versions from the current).

Reply:
So how do you suggest to handle this? With warehouses and stocks this field will blow up and returns an error.

Reply:
We need to provide some workaround to return here a meaningful value. E.g. do we have the concept of "allocated quantity" with warehouses? I guess we do but now it's kept at the stock level? Are we able to calculate the sum of allocated quantities for a single variant in all stocks? That could be one solution.

Reply:
@maarcingebala I will think about this after testing current changes as everything might change after that :) There is allocated quantity in stocks, but I will need to check where this field is used as for storefront we can define quantity allocated by country, but for the dashboard, we would need the sum